meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -43,6 +68,14 @@ do_install() {
         systemd_unitdir=${systemd_unitdir} \
         install-bin \
         install-systemd
+
+    # install configuration
+    mkdir -p  ${D}/${sysconfdir}/mender
+    if [ -f ${WORKDIR}/mender-shell.conf ]; then
+        install -m 0600 ${WORKDIR}/mender-shell.conf ${D}/${sysconfdir}/mender/mender-shell.conf
+    else
+        install -m 0600 ${B}/mender-shell.conf ${D}/${sysconfdir}/mender/mender-shell.conf
```

Comment:
Oh, I see, you choose one or the other. I think it's better to merge though, do you agree?

Reply:
Yes, it is a better solution. For example the user might have other fields, but missing the server.

I am reworking, thanks!